infra/cifuzz/actions/action.yml

```diff
@@ -9,9 +9,13 @@ inputs:
     description: 'The total time allotted for fuzzing in seconds.'
     required: true
     default: 360
+  failure-allowed:
+    description: 'Set when action should be run in no fail mode.'
```

Comment:
As I read this, there is a mismatch between the logic implied by the name and the description:
- failure allowed
- run in no fail mode

I'd propose to call it `dry-run`, I think it's kinda standard for such situations. The comment will then be `If set, run the action without actually reporting a failure` or something like that. The default value should be `False` then

Reply:
Switched.